kiwi/builder/container.py

```diff
@@ -57,6 +60,23 @@ def __init__(self, xml_state, target_dir, root_dir):
         self.target_dir = target_dir
         self.container_config = xml_state.get_container_config()
         self.requested_container_type = xml_state.get_build_type_name()
+        self.base_image_uri = xml_state.get_derived_from_image_uri()
+        self.base_image = None
+
+        if self.base_image_uri:
+            base_image_origin = self.base_image_uri.translate()
+
+            # we expect the base image to be unpacked by the kiwi prepare step
+            # and stored inside of the root_dir/image directory
+            self.base_image = os.path.normpath(
+                os.sep.join(
+                    [
+                        root_dir, 'image',
+                        base_image_origin.replace('.tar.xz', '')
```

Comment:
ah ok, so the prepare step always leaves the same file and that is a fix name we can rely on ? If that's the case I'm ok with it. But I would prefer if that name will be part of a method in the Defaults class, including the mentioned comment. Any code dealing with the file should take the name from this unique place. Would that be ok ?